docs/maintainers/maintainers-guide.md

```diff
@@ -0,0 +1,44 @@
+---
+id: maintainers-guide
+title: "@open-sauced/maintainers-guide"
+sidebar_label: "@open-sauced/maintainers-guide"
+keywords:
+  - "@open-sauced/maintainers-guide"
+---
+
+Welcome to OpenSauced, Maintainers Guide for contributors who are interested in being part of the maintainer team for OpenSauced repositories. 
+
+At OpenSauced, we're empowering contributors to work in the open and supporting maintainers to make data-driven decisions for their projects.
+
+It's important to us that we maintain a healthy environment for contributors and maintainers of our projects.
+
+## How do I Join the Maintainers Team?
+- Sign up for [opensauced.pizza](https://opensauced.pizza/)
+- Join discord.
+- Prove your pizza worth!
+
+## What does the Maintainers Team do?
+Well, like all things in tech, it depends. We have different maintainer teams for each of the repositories, so it's important that you communicate with the admin maintainer if you have any questions.
+
+### Core Responsibilities
+Your responsibilities may include:
+- Reviewing and merging pull requests (PRs).
+- Providing comments and responses on PRs, issues, discussions, and Discord messages.
+- [Triage issues](../contributing/triage-guide.md) and bug reports.
+- Maintaining project documentation.
+- Collaborating with the community. 
+
+**Think of creating a positive space for contributors at all stages as one of your most important responsibilities.**
+
+As a maintainer, you must:
+- Adhere to our [Code of Conduct](../contributing/code-of-conduct.md) and be an example for treating contributors with respect.
+- Communicate with the other maintainers in a timely and reasonable manner.
+- Understand and adhere to project standards.
+
+## Committing and Merging Changes
+1. For commit and pull request standards, please refer to [introduction to contributing](../contributing/introduction-to-contributing.md).
+2. Before merging in changes, be sure to run the project locally if UI changes were made.
+3. **Squash and merge commits** when you merge in a PR.
+
+Thank you for your interest in becoming a maintainer! Please reach out if you need help, guidance, or clarification at any time.
```

Comment:
[question/suggestion] Who can people contact if they are interested in being a maintainer? Consider adding this information to make the CTA clearer.